//PYTHON BUILT IN EXCEPTIONS

>>> if a<3:
...     print(a)
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'a' is not defined
>>> 
>>> a=2
>>> if a<3:
...     print(a)
... 
2
>>> 
>>> if a<3
  File "<stdin>", line 1
    if a<3
         ^
SyntaxError: invalid syntax
>>> 
>>> 1/0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: division by zero
>>> 
>>> open('test.txt')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'test.txt'
>>> 

